Because gtk-2/gtk is linked against gtk-2/types, types must be loaded
first in order that gtk's symbols will be resolve at runtime.
(Compare to gtk2-perl bindings.)

--- origsrc/rep-gtk-0.18.3/examples/clist.jl	2002-10-25 01:14:06.000000000 -0500
+++ src/rep-gtk-0.18.3/examples/clist.jl	2008-11-27 02:45:54.516062800 -0600
@@ -6,6 +6,7 @@
 
     (open rep
 	  rep.system
+	  gui.gtk-2.types
 	  gui.gtk-2.gtk)
 
   (define titles ["name" "uid" "gid" "passwd" "gecos" "home" "shell"])
--- origsrc/rep-gtk-0.18.3/examples/hello.jl	2002-10-25 01:14:06.000000000 -0500
+++ src/rep-gtk-0.18.3/examples/hello.jl	2008-11-27 02:46:58.589090200 -0600
@@ -8,6 +8,7 @@
 
     (open rep
 	  rep.system
+	  gui.gtk-2.types
 	  gui.gtk-2.gtk)
 
   (define window (gtk-window-new 'toplevel))
--- origsrc/rep-gtk-0.18.3/examples/status-icon	2008-07-31 14:53:33.000000000 -0500
+++ src/rep-gtk-0.18.3/examples/status-icon	2008-11-27 02:46:32.746522600 -0600
@@ -8,6 +8,7 @@
 
     (open rep
           rep.system
+          gui.gtk-2.types
           gui.gtk-2.gtk)
 
   (define window (gtk-window-new 'toplevel))
--- origsrc/rep-gtk-0.18.3/examples/text.jl	2002-10-25 01:14:06.000000000 -0500
+++ src/rep-gtk-0.18.3/examples/text.jl	2008-11-27 02:35:53.089438600 -0600
@@ -8,6 +8,7 @@
 
     (open rep
 	  rep.system
+	  gui.gtk-2.types
 	  gui.gtk-2.gtk)
 
   (define window (gtk-window-new 'toplevel))
